### PR TITLE
Eng 18630 topic restore

### PR DESCRIPTION
**Pull request by n3world**
Topic data is kept after a restore so it needs to be treated the same way an export stream is treated during recovery.

ExecutionEngine.exportAction was always called with true as the first argument. That means that it was only used to set the export stream positions. Remove the boolean argument and rename the method to setExportStreamPositions. Also, remove any extra code for old use including a return value.

Related https://github.com/VoltDB/pro/pull/3075

### CODE REVIEW

File: src/ee/voltdbipc.cpp [hunk 1]
```diff
@@ -1616,21 +1615,19 @@ void VoltDBIPC::tableHashCode( struct ipc_command *cmd) {
     writeOrDie(m_fd, (unsigned char*)response, 9);
 }
 
-void VoltDBIPC::exportAction(struct ipc_command *cmd) {
+void VoltDBIPC::setExportStreamPositions(struct ipc_command *cmd) {
     export_action *action = (export_action*)cmd;
```
[the remark this answers is not in the document]
Comment (n3world) [reply src/ee/voltdbipc.cpp]: Missed that I can do that too

File: src/ee/execution/VoltDBEngine.h [hunk 2]
```diff
@@ -526,18 +526,15 @@ class __attribute__((visibility("default"))) VoltDBEngine {
                 ReferenceSerializeInputBE& serializeIn, std::vector<int>& retPositions);
 
         /**
-         * Perform an action on behalf of Export.
+         * Set the export stream positions for the given stream
          *
-         * @param syncAction if syncAction is true, the stream offset being set for a table
          * @param ackOffset the reference to the USO of the next row inserted in the stream
```
Comment (rdykiel) [inline src/ee/execution/VoltDBEngine.h]: Please make sure the comments are in sync

File: src/ee/voltdbjni.cpp [hunk 3]
```diff
@@ -1202,24 +1202,19 @@ SHAREDLIB_JNIEXPORT jlong JNICALL Java_org_voltdb_jni_ExecutionEngine_nativeTabl
...
- * any error this will be less than 0.  For any call with no
- * pollAction, any value >= 0 may be ignored.
  */
-SHAREDLIB_JNIEXPORT jlong JNICALL Java_org_voltdb_jni_ExecutionEngine_nativeExportAction
+SHAREDLIB_JNIEXPORT void JNICALL Java_org_voltdb_jni_ExecutionEngine_nativeSetExportStreamPositions
   (JNIEnv *env,
    jobject obj,
    jlong engine_ptr,
-   jboolean syncAction,
    jlong ackOffset,
```
Comment (rdykiel) [inline src/ee/voltdbjni.cpp]: ackOffset

Comment (n3world) [reply src/ee/voltdbjni.cpp:1218]: I didn't do anything to remove ackOffset so that name still exists everywhere it did before.

**Pull request merged**